### DISCUSSION TITLE
HTC/DetectoRS coco_instance_semantic.py--stuffthings

### DISCUSSION THREAD

**Discussion by DLUT-LYZ** [open-mmlab/mmdetection#9102] [Q&A] [answered]
### 📚 The doc issue

我想问一下datasets/coco_instance.py与datasets/coco_instance_semantic.py两个文件所对应的任务有什么区别？我认为都是实例分割任务，对于coco_instance_semantic.py中的stuffthingmaps/train2017/起到什么作用在HTC/DetectoRS中

**Comment by hhaAndroid** [comment, marked as answer]
@lllyyqq The only difference is that `coco_instance_semantic` will additionally load semantic segmentation labels. For some instance segmentation algorithms, semantic segmentation branch assistance may be required.

**Comment by DLUT-LYZ** [comment]
OK, I see.Thanks！！
